src/routers/userRouter.ts

```diff
@@ -202,10 +202,10 @@ router.post("/login", verifyReq(loginSchema), async (req, res) => {
         await user.save();
     }
 
+    res.json(await sanitizeUser(user));
```

Comment:
Because the X3DH parameters were cleared before user sanitization, the response didn't have them.